handbook/product/README.md

```diff
@@ -209,6 +209,25 @@ When writing copy, consider whether additional information is necessary before a
 
 When writing the time in the UI using "am" and "pm" abbreviations, write them **without space** between time and abbreviation, with **no punctuation**, and use **lowercase** letters (e.g. Working hours are 8am to 5pm).
 
+**Writing error messages**
+
+When writing error messages in the UI or CLI, follow these rules:
+- If the solution to the error isn't obvious, write a message with the **error** followed by the **solution**. For example, "No hosts targeted. Make sure you provide a valid hostname, UUID, osquery host ID, or node key."
+- If the solution is obvious when additional info is provided, write a message with the **error** followed by **additional info**. For example, "You don’t have permission to run the script. Only users with the maintainer role and above can run scripts."
+
+**Fleetctl commands with `--hosts` or `--host` flag**
```

Comment:
I think the first part about how to structure the error messages is helpful, but I'm sort of wondering if this part about `--hosts`/`--host` might be a bit too specific (i.e. harder to accurately maintain) for the product handbook page. I'd be more likely to give new designers the advice: try out the area of the CLI you're designing for, then follow your best judgement about whether to deviate from its current conventions (since, from what I've seen, different commands sometimes respond differently in the success/error states).

That said, you may have done a deeper dive into fleetctl than I have at this point, so if you feel confident in this, plz ignore me!